Makefile.in

```diff
@@ -4,19 +4,23 @@ AS		?= $(CC)
 AR		= ar
 RM		= rm -f
 
-CFLAGS	+= -Wall -Werror -Wextra -fpic -fno-builtin -fno-stack-protector -nostdlib \
-			 -nostdinc -Iinclude
+BASE_CFLAGS	+= -Wall -Werror -Wextra
+LIBC_CFLAGS = $(BASE_CFLAGS) -Iinclude -nostdlib -nostdinc -fno-stack-protector -fno-builtin -fpic
+SYSDEPS_CFLAGS += $(BASE_CFLAGS) -fno-stack-protector -fpic
```

Comment:
Shouldn't it be a simple `=` and not a `+=` ? 
But I think it doesn't really matter.